### ISSUE TITLE
Fullscreen on mobile effected by ID in Properties

### ISSUE THREAD

**Issue by lumicreative** [Scirra/Construct-3-bugs#2092]
## Problem description

Changing the ID in properties effects fullscreen on mobile, resulting in black bars on the top and bottom. I have two version for the c3p.

## Attach a .c3p

[ID-YesFullscreen.zip](https://github.com/Scirra/Construct-3-bugs/files/2431154/ID-YesFullscreen.zip)

[ID-NoFullscreen.zip](https://github.com/Scirra/Construct-3-bugs/files/2431153/ID-NoFullscreen.zip)

## Steps to reproduce

1. Export ID-YesFullscreen.c3p as Debug APK 
2. Install APK on mobile device (preferably Note 8 or tall viewport size)
3. Run the installed app
4. Observe no issues with full screen
5. Rename the ID to "com.beastattack.c1434846484727"
6. Re-export as Debug APK
7. Install APK on mobile device (preferably Note 8 or tall viewport size)
8. Run the installed app
9. Observe black bars on top and bottom

## Observed result

Black bars on appear on top and bottom on mobile phone

## Expected result

Full screen should show even though I change the ID in properties. Strange that it has an effect on fullscreen.

## Affected browsers

* Chrome: Yes
* Firefox: N/A
* Edge: N/A
* Safari: N/A

## System details

<details><summary>View details</summary>

Platform information
Browser: Chrome
Browser version: 69.0.3497.100
Browser engine: Blink
Browser architecture: (unknown)
Context: browser
Operating system: Mac OS X
Operating system version: 10.13_6
Operating system architecture: (unknown)
Device type: desktop
Device pixel ratio: 2
Logical CPU cores: 8
Approx. device memory: 8 GB
User agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/69.0.3497.100 Safari/537.36
C3 release: r120 (stable)
Language setting: en-US

Local storage
Storage quota (approx): 62 gb
Storage usage (approx): 369 mb (0.6%)
Persistant storage: No

Browser support notes
This list contains missing features that are not required, but could improve performance or user experience if supported.

Nothing is missing. Everything is OK!
WebGL information
Version string: WebGL 2.0 (OpenGL ES 3.0 Chromium)
Numeric version: 2
Supports NPOT textures: yes
Supports GPU profiling: no
Vendor: ATI Technologies Inc.
Renderer: AMD Radeon Pro 560 OpenGL Engine
Major performance caveat: no
Maximum texture size: 16384
Point size range: 1 to 2048
Extensions:

EXT_color_buffer_float
EXT_texture_filter_anisotropic
OES_texture_float_linear
WEBGL_compressed_texture_s3tc
WEBGL_compressed_texture_s3tc_srgb
WEBGL_debug_renderer_info
WEBGL_debug_shaders
WEBGL_lose_context
Audio information
System sample rate: 44100 Hz
Output channels: 2
Output interpretation: speakers
Supported decode formats:

Ogg Opus (audio/ogg; codecs=opus)
WebM Opus (audio/webm; codecs=opus)
Ogg Vorbis (audio/ogg; codecs=vorbis)
WebM Vorbis (audio/webm; codecs=vorbis)
MPEG-4 AAC (audio/mp4; codecs=mp4a.40.5)
MP3 (audio/mpeg)
FLAC (audio/flac)
PCM WAV (audio/wav; codecs=1)
Supported encode formats:

WebM Opus (audio/webm; codecs=opus)
Video information
Supported decode formats:

WebM VP9 (video/webm; codecs=vp9)
WebM VP8 (video/webm; codecs=vp8)
Ogg Theora (video/ogg; codecs=theora)
H.264 (video/mp4; codecs=avc1.42E01E)
Supported encode formats:

WebM VP9 (video/webm; codecs=vp9)
WebM VP8 (video/webm; codecs=vp8)

</details>

**Comment by lumicreative** [comment]
Thanks for your help on this @shortercode , any updates? Thanks!

**Comment by lumicreative** [comment]
@shortercode Another example as well:

**No full screen:** com.lumicreative.kunkong
**Yes full screen:** com.rolling.kunkong

**Comment by shortercode** [comment]
Going to be honest, this is a real confusing issue.

To remove some of the variables from the issue I exported to a cordova project, then I've been creating Android studio projects from that with different package ID values. I've done the majority of my testing with "hide statusbar" disabled, which is effectively cordova's "fullscreen" option ( on Android anyway ). It wasn't clear what option you had chosen, but there's still noticeable different behaviour with it disabled ( the nav bar is always visible for the "non fullscreen" IDs). 

So I've spent a fair chunk of time looking at these ( several hours ) and I have no idea why they behave differently, the projects are exactly the same other than the ID values. I've debugged the start up code and there's no difference. There is no obvious pattern to the IDs that are not fullscreen; `com.beastattack.c1434846484727` isn't fullscreen but `com.beastattack.c1434846484728` and `com.beastattack.c143484648472` are.

The only hint of an issue I can find is that the samsung game service seems to fire into life with the non-fullscreen one and some extra controls appear in the nav bar for the game service. I can't be sure if it's gametools showing the nav bar, or gametools being active because of the nav bar, but we can assume that it's related.

Unfortunately I can't debug the game service, the only reason I know that it's doing something is that it spits a load of debug info into the application log. The following log is enlightening

`2018-11-19 18:23:38.932 12257-12257/? I/GameTools: GameToolsService : onGameResume appinfo loading completed. com.beastattack.c1434846484727 FullscreenTest 12768 0 0.0, mWideScreenMode=0 misGameApp=true`

So it's identifying it as a game... for some reason. My best guess is that the specific package ID has been registered on a service somewhere as a game. Beyond this I don't think I can help, I can't see anything we are doing wrong for us to fix!
